modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -475,8 +476,12 @@ const setup = (editor: Editor, setupForTheme: ThemeRenderSetup): RenderInfo => {
       OuterContainer.focusToolbar(outerContainer);
     });
 
-    editor.addCommand('ToggleToolbarDrawer', (_ui, options?: { skipFocus: boolean }) => {
+    editor.addCommand('ToggleToolbarDrawer', (_ui, options?: { skipFocus: boolean }, args?: ExecCommandArgs) => {
       if (options?.skipFocus) {
+        // eslint-disable-next-line no-console
+        console.warn('ToggleToolbarDrawer skipFocus is deprecated see migration guide: ....');
```

Comment:
I forgot to mention this on #10281 but I'd prefer these messages go in `Deprecations.ts`. It makes them easier to find later.

It doesn't have to be complicated, just move the whole line into a function in Deprecations and call it from here.

Reply:
Also we might need to have a follup up ticket if we want to merge this without having a URL to the migration guide. That could then be a variable the the `Deprecations.ts` file if you move this log into that.